BookingsAPI/Bookings.UnitTests/Bookings.UnitTests.csproj

```diff
@@ -19,4 +19,7 @@
     <ProjectReference Include="..\Bookings.Domain\Bookings.Domain.csproj" />
     <ProjectReference Include="..\Testing.Common\Testing.Common.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This usually appears if the folder is empty. Could you remove the folder or remove the unncessary entry here please. Thanks.